app-web/plugins/gatsby-source-github-all/__tests__/sourceNodes.itest.js

```diff
@@ -20,6 +20,14 @@ describe('Integration Tests Source Nodes', () => {
     shortid.generate = jest.fn(() => 1);
   });
 
+  test('validateRegistryItem returns true if name and sourceProperties exist', () => {
+    const registryItem = {
+      name: 'foo',
+      sourceProperties: {},
+    };
+    expect(validateRegistryItem(registryItem)).toBe(true);
```

Comment:
maybe also test for false?

Reply:
good point:P I think i added the false test case in pr #275 ...i dont know i just work here